settings.php

```diff
@@ -37,6 +37,15 @@
 $config_directories['active'] = 'files/config_' . md5($database) . '/active';
 $config_directories['staging'] = 'files/config_' . md5($database) . '/staging';
 
+/**
+ * Skip the configuration staging directory cleanup
+ *
+ * For some development workflows it may be necessary to not delete the
+ * config staging files each configuration sync, such as when the
+ * configuration files in version control.
```

Comment:
`config staging files` -> `staging config files`

There's also some words missing here, which make this hard to read/understand:

- ...staging files **with/after** each configuration sync...
- ...when the configuration files **are** in version control

How about something like this?:

```suggestion
 * Some development workflows, may require retaining the staging config files
 * after each configuration sync (i.e. when the configuration files are in
 * version control).
```

Reply:
Changing this to a config variable so won't add anything to settings file.